sql/parser/alter.go

```diff
@@ -38,6 +38,10 @@ func (p *Parser) parseAlterTableAddFieldStatement(tableName string) (_ query.Alt
 		return stmt, err
 	}
 
+	if stmt.Constraint.IsPrimaryKey {
+		return stmt, &ParseError{Message: "Adding a field with primary key constraint is not supported now."}
```

Comment:
This is more consistent with other errors in this package and I am not sure that we'll even support this in the future.
```suggestion
		return stmt, &ParseError{Message: "cannot add a PRIMARY KEY constraint"}
```